packages/hdwallet-ledger/src/bitcoin.ts

```diff
@@ -143,7 +143,7 @@ export async function btcSignTx(
 ): Promise<core.BTCSignedTx> {
   const supportsSecureTransfer = await wallet.btcSupportsSecureTransfer();
   const slip44 = core.mustBeDefined(core.slip44ByCoin(msg.coin));
-  const txBuilder = new bitcoin.TransactionBuilder(networksUtil[slip44].bitcoinjs as any);
+  const psbt = new bitcoin.Psbt({ network: networksUtil[slip44].bitcoinjs as bitcoin.Network });
```

Comment:
`TransactionBuilder` class is gone in profit of `Psbt`:
https://github.com/bitcoinjs/bitcoinjs-lib/blob/32e08aa57f6a023e995d8c4f0c9fbdc5f11d1fa0/CHANGELOG.md?plain=1#L55